dags/inference/maxtext_inference.py

```diff
@@ -457,7 +457,7 @@
           "quant_mode": W_INT8_KV_INT8,
           "quantization": "int8",
           "quantize_kvcache": "true",
-          "per_device_batch_size": 258,
```

Comment:
Wait! we have already quantized for inference on MaxText?

Reply:
Yes!

Reply:
Which implementation is this one? I don't think I enabled quantization for both matmul or megablox yet (at least for the MoE block). Or other implementation we are talking about here?